second_response_to_reviewers.tex

```diff
@@ -0,0 +1,366 @@
+not
+allowed in the authors’ simulation so no other strategies can be used. How
+useful then
+is it to know some other strategy C can successfully invade strategy B if
+strategy C
+isn’t available?
+\end{verbatim}
+
+This is more useful than the already published (and heavily cited) works in this
+area that look at the score of A against B as opposed to the ability of A to
+invade/defend B.
+
+\begin{verbatim}
+How useful is it knowing that strategy A can or cannot resist invasion by
+strategy
+B when players rarely know exactly what strategy their opponent is using? In
+par-
+ticular, before the game begins players won’t know what strategy others intend
+to
+use.
+\end{verbatim}
+
+This feels like a question about the entirety of the research field on the IPD.
+
+We refer the referee to the vast quantity of literature in this field and in
+particular highlight that one of the main goals of this area are to understand
+properties of environment and strategies that enable for emergence of a given
+type of behaviour. In the case of this work, we show that in evolutionary
+environments, handshake mechanisms can emerge which contribute to the emergence
+of cooperation.
+
+We have added a sentence to the paper further explaining how this work offers a
+more evolutionary robust version of Robert Axelrod's original tournament.
+
+\section{Comments from third reviewer}
+
+\begin{verbatim}
+    I saw no significant contribution in the field of evolutionary computation
+    itself. The whole paper would have been a great *experimental section* after
+    some new idea or scheme.
+\end{verbatim}
+
+We have justified the contribution in many ways but agree that this paper is a
+strong numerical experimental paper (as evidenced by the title) following the
+novel idea of creating tournaments with Moran processes as ways to score pairs of
+strategies.
+
+\begin{verbatim}
+Indeed, results appears sensible, when not obvious. Collective strategies
+perform well, the usual suspects are hard to invade. However, generally
+speaking, the main problem is that with IPD is extremely hard to infer
+meaningful data from the mere statistical results, as it would be hard to draw
+conclusions about the "best opening" looking at students playing chess. Indeed,
+such an analysis makes perfect sense as soon as one restricts the domain to a
+specific grandmaster, or a specific top-level tournament.
+\end{verbatim}
+
+A lot of the original and contemporary research in this area uses the IPD to
+show why/how
+cooperative behaviour may evolve. This has often been through large numerical
+analyses such as these. However, none of them (as discussed in the literature
+review) consider Moran processes on this scale. Furthermore, one of our main
+conclusions/findings is that handshake mechanisms can also appear through
+evolutionary processes and in particular that these can explain the emergence of
+cooperation.
```

Comment:
Let's add that this is directly analogous to "the best opening" in chess. Moreover, our numerical results call into question several common assumptions in the literature, e.g. that it's enough to only consider memory-one strategies (it's demonstrable not the case), that ZD strategies are the best yet they don't win here, etc. This work specifically identifies fruitful and valuable new directions for future work.

Reply:
Let's also add that we show that "collective strategies" directly emerge from evolutionary processes and are reinforced by standard evolutionary mechanisms, which is a major theme of the paper and very relevant to the study of the evolution of cooperation.